i18n/fr/docusaurus-plugin-content-docs/current/full-node/Full-Node-Starter-Guide.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position : 10
+titre : Guide complet de démarrage du fullnode
```

Comment:
Still needs to be changed.

Reply:
fixed in https://github.com/alephium/wiki/pull/39/commits/aa98c0da5a79c88cbe3ac669aeaa6acb4a6c3f8f